app/scripts/components/common/layout-root/useFeedbackModal.ts

```diff
@@ -0,0 +1,22 @@
+import { useContext, useCallback } from "react";
+import { LayoutRootContext } from "./context";
+
+/**
+ * Hook to access the feedback modal.
+ */
+export function useFeedbackModal() {
```

Comment:
Flagging that GHG instance is using this hook https://github.com/US-GHG-Center/veda-config-ghg/blob/develop/overrides/development/component.tsx#L9 , expecting it to be in `layout-root`